### ISSUE TITLE
Add CSV output?

### ISSUE THREAD

**Issue by KitClement** [jonatanklosko/groupifier#1]
When I create groups manually, I use my raw data for many purposes like posting to a table on the WCA website or to mail merge into a custom nametag format we use. It seems a bit silly, but I feel like getting the raw data used to create the PDFs could be of use.

**Comment by jonatanklosko** [comment]
The final plan is to save group assignments to the WCA website, so that it can be used anywhere else. Then you would be able to use WCIF JSON. However, for this to happen we need a way of taking CSV files generated by external registration systems and creating WCA registrations in the database without the need for accounts. Once this is done, Groupifier will be able to always manage those registrations and save all the informations back to the WCA.

**Comment by KitClement** [comment]
Ah, got it, that sounds very cool!

The one thing that I would still be missing if I were to use this is to have the data for my custom name tags in MS Publisher. I use a mail merge on a CSV file with all of the groups/jobs for all competitors. I'm not too familiar with converting JSON formats, but it sounds like this could be converted to a CSV as needed.

**Comment by jonatanklosko** [comment]
Converting WCIF JSON into a CSV file shouldn't be more difficult than doing so with the data I build =)

**Comment by Mollerz** [comment]
Adding to this, if we could get the "Summary" to output in markdown so we can straight copy and paste it into a tab in the WCA competition tabs that would be _sweet_.

**Comment by jonatanklosko** [comment]
I've just added an option to download people data/assignments into a JSON file. I felt like it would be hard to create a reasonable CSV file for this kind of data structure.

In the future WCIF will certainly used for dealing groups info, so any kind of export will no longer make sense.
